packages/open-ui/src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from 'react';
+import { useAnimate } from 'react-simple-animate';
+import ReactDOM from 'react-dom';
+import classNames from 'classnames';
+import ScrollLock from 'react-scrolllock';
+
+import { Title } from 'components/Typography';
+import { Button } from 'components/Button';
+import { IntentOption } from 'interfaces/IntentOption';
+import { toSentenceCase } from 'utils/strings/to-sentence-case';
+import { CloseButton } from './CloseButton';
+import styles from './Modal.module.scss';
+
+interface ButtonProps {
+  onClick: () => void;
+  text: string;
+}
+
+interface ModalProps {
+  active: boolean;
+  close: () => void;
+  intent?: IntentOption;
+  primaryButton?: ButtonProps;
+  secondaryButton?: ButtonProps;
+  showCloseButton?: boolean;
```

Comment:
@remotealex do we need this prop bro? like if we have secondaryCloseButton we gonna display it else not ?

Reply:
I think some people might not want to close button. Maybe we can have it enabled by defauly and change it to `hide`??

Reply:
@remotealex yeah but current execution of code is like if we have secondaryButtonProp passed only then our button is showing

Reply:
`closeButton` is the (X) in the top right corner...?

Reply:
ahh :( my bad I was mentioning about hte secondaryButton ( as it's was used as close ).